packages/terra-framework-docs/src/terra-dev-site/doc/collapsible-menu-view/example/AlwaysCollapsedMenuItemsDemo.jsx

```diff
@@ -1,30 +1,19 @@
 import React from 'react';
+import IconCaretDown from 'terra-icon/lib/icon/IconCaretDown';
 import IconTable from 'terra-icon/lib/icon/IconTable';
-import IconFlowsheet from 'terra-icon/lib/icon/IconFlowsheet';
+import IconTreemap from 'terra-icon/lib/icon/IconTreemap';
```

Comment:
Since we are updating these, let's fix them to use the named exports introduced in https://github.com/cerner/terra-core/pull/3915.

Reply:
Updated here: https://github.com/cerner/terra-framework/pull/1798/commits/1767644f1ec11f76142b473322a67ff74d837e81

Reply:
Thanks for making these changes!